lib/improver/utilities/ancillary_creation.py

```diff
@@ -225,9 +225,16 @@ class SaturatedVapourPressureTable(object):
     Plugin to create a saturated vapour pressure lookup table.
     """
 
-    def __init__(self, t_min=183.15, t_max=338.15, t_increment=0.1):
+    def __init__(self, t_min=183.15, t_max=338.25, t_increment=0.1):
         """
-        Initialise class.
+        Create a table of saturated vapour pressures that can be interpolated
+        through to obtain an SVP value for any temperature within the range
```

Comment:
I'm afraid not. It's about the sound, not the letter. SVP starts with the vowel sound /ess/ so it is preceded by "an".

Reply:
Depends if you read it as 'SVP' or convert the acronym into 'Saturation Vapour Pressure' as you read it. In the former, 'an' is more appropriate, in the latter I would argue 'a' is more appropriate. I tend to convert it to the words rather than the acronym, so tend to favour the latter. Happy to leave it as it is.